integrations/docker/images/stage-2/chip-build-java/Dockerfile

```diff
@@ -1,12 +1,12 @@
-ARG VERSION=1
+ARG VERSION=2
 FROM ghcr.io/project-chip/chip-build:${VERSION}
 LABEL org.opencontainers.image.source https://github.com/project-chip/connectedhomeip
 
 # Ensure some java is installed
 RUN set -x \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
-    openjdk-8-jdk \
```

Comment:
changes to dockerfile need to update the version of images in chip-build (because chip-build is the base, we share that version with all. This is easier to manage than trying to have each docker file maintain its own version)